cocos/audio/CMakeLists.txt

```diff
@@ -3,14 +3,12 @@ set(COCOS_AUDIO_SRC
     audio/AudioEngine.cpp
     )
 
-if(WINDOWS)
+if(LINUX OR WINDOWS)
     set(COCOS_AUDIO_PLATFORM_SRC
-        audio/win32/SimpleAudioEngine.cpp
-        audio/win32/MciPlayer.cpp
-        audio/win32/MciPlayer.h
-        audio/win32/AudioEngine-win32.cpp
```

Comment:
it seems that `MciPlayer.cpp` was removed from windows. was this done on purpose? or is it just a copy&paste error?